src/System.Security.Cryptography.X509Certificates/tests/project.json

```diff
@@ -23,13 +24,10 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00520-02"
   },
   "frameworks": {
-    "netstandard1.4": {}
+    "netstandard1.6": {}
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```

Comment:
This probably means we need to split the build.  We'd like to have netcore50 tests run.